test/common/http/http1/codec_impl_test.cc

```diff
@@ -69,6 +71,65 @@ TEST_F(Http1ServerConnectionImplTest, Http10) {
   EXPECT_EQ(Protocol::Http10, codec_->protocol());
 }
 
+TEST_F(Http1ServerConnectionImplTest, Http11AbsolutePath) {
+  InSequence sequence;
+
+  //There's probably a better way to do this....
```

Comment:
To do which?
If nothing else if you're going to have a lot of copy-paste boilerplate it'd be nice to just stick it in test set up (not something super common in the current code base but I'm trying to do more of this as we add more boilerplate tests) or do a test function like 
VerifyExpectedHeaders("GET http://www.somewhere.com/ HTTP/1.1\r\nHost: bah\r\n\r\n",
                                       TestHeaderMapImpl{{":authority", "www.somewhere.com"}, {":path", "/"}, {":method", "GET"}})